src/middleware/packages/core/service.js

```diff
@@ -148,6 +147,15 @@ const CoreService = {
       });
     }
 
+    if (this.settings.vc !== false) {
+      this.broker.createService({
+        mixins: [VCService],
+        settings: {
+          ...this.settings.vc
+        }
+      });
+    }
```

Comment:
The CoreService only include the minimum services to run a LDP+ActivtiyPub server.
We shouldn't include VC unless it is required by other services (but that shouldn't be the case).